.gitignore

```diff
@@ -14,6 +14,7 @@
 /var/cache/*
 /var/logs/*
 /var/sessions/*
+/var/SymfonyRequirements.php
```

Comment:
I just read that it was better to let it in the tracked files list :
https://stackoverflow.com/questions/35315225/why-is-symfonyrequirements-php-excluded-from-gitignore